### ISSUE TITLE
REGEX error on some Spanish Bio Codes

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/robotoff#53]
`en:es-eco-023` instead of `en:es-eco-023-ma`
https://world.openfoodfacts.org/product/8435177051395/amandes-cranberries-et-raisins-secs-auchan
`\nES-ECD-023 MA\n` 
https://static.openfoodfacts.org/images/products/843/517/705/1395/1.json

**Comment by teolemon** [comment]
They already are in the taxonomy
``` 
<en:EU Organic
en:ES-ECO-023-MA
de:ES-ECO-023-MA
pt:ES-ECO-023-MA
country:Spain

<en:EU Organic
en:ES-ECO-024-MU
de:ES-ECO-024-MU
pt:ES-ECO-024-MU
country:Spain

```

**Comment by hangy** [comment]
https://github.com/openfoodfacts/robotoff/blob/e5534b488a988c23cc2c56fbf59d01384d7130c5/robotoff/insights/ocr/label.py#L38

Shouldn't match `ES-ECO-023` labels (without `-MA` suffix), because the two-letter alphanumeric suffix is required. There is no need to create an insight for this, if the full pattern was not found.

**Comment by raphael0202** [comment]
Fixed in a27b9b620dd46bf929f4acfcb14da60a098bb1bc
